Unity-Technologies/ScriptableRenderPipeline
# Remove ability to create Preset from Shader Graph and Sub Graph assets

Got a bug that Presets made from SG files are broken. https://fogbugz.unity3d.com/f/cases/1220914/ But what does an SG preset even mean? Easy enough to opt out. No public docs but internal doc is https://docs.google.com/document/d/1Jg8qZUGhu92eeSFV8as5L7DHdDxloVGLJYhdqEr9J7g/edit#heading=h.oduge3518o4p


## Checklist for PR maker

---
### Testing status

**Manual Tests**: What did you do?
- Verified that Preset creation button is hidden for .shadergraph and .shadersubgraph files in the inspector. See SG inspector on top, no little slider icon vs Material inspector below with little slider icon.
![Screen Shot 2020-02-27 at 12 26 11 PM](https://user-images.githubusercontent.com/21229526/75483862-85826500-595c-11ea-9e38-99977a186f94.png)

Also note that any SG Preset files you've created already will give the error "Unable to load this Preset, the type is not supported." This is expected.

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline
